requirements/edx/custom.txt

```diff
@@ -2,7 +2,7 @@
 
 # When updating a hash of an XBlock that uses xblock-utils, please update its version hash in github.txt.
 -e git+https://github.com/edx-solutions/xblock-mentoring.git@8837eb5d91fed05ec4758dfd9b9e7adc5c906210#egg=xblock-mentoring
--e git+https://github.com/edx-solutions/xblock-image-explorer.git@v1.1.2#egg=xblock-image-explorer==1.1.2
+-e git+https://github.com/edx-solutions/xblock-image-explorer.git@v1.1.2#egg=xblock-image-explorer==1.1.3
```

Comment:
This is not bumping up the actual version (i.e. git tag)

Reply:
Oh, sorry about that. I've fixed it now.